master.go

```diff
@@ -124,10 +124,9 @@ func newMasterCollector(httpClient *httpClient) prometheus.Collector {
 			return nil
 		},
 		gauge("master", "slaves_state", "Current number of slaves known to the master per connection and registration state.", "state"): func(m metricMap, c prometheus.Collector) error {
-			active := m["master/slaves_active"]
-			inactive := m["master/slaves_inactive"]
-			disconnected := m["master/slaves_disconnected"]
-			connected := m["master/slaves_connected"]
+			active, ok := m["master/slaves_active"]
+			inactive, ok := m["master/slaves_inactive"]
+			disconnected, ok := m["master/slaves_disconnected"]
```

Comment:
@elubow these `ok` values are re-assigned each time so you'll only error if `master/slaves_disconnected` is unset, `slaves_active` and `slaves_inactive` are ignored. I've explicitly checked them inline which is rather cumbersome. I might clean this up in the future.